File: python/smoke.py
"""Quick client smoke — polite delay on."""

from __future__ import annotations

import json

from lk_flood_api_docs import LkFloodApiDocsClient


def json_preview(data: object, limit: int = 200) -> str:
    try:
        return json.dumps(data, ensure_ascii=False)[:limit]
    except TypeError:
        return str(data)[:limit]


with LkFloodApiDocsClient(default_delay_seconds=1.0) as client:
    print("smoke", client.slug, "->", "flood_latest")
    data = client.flood_latest()
    preview = data[:200] if isinstance(data, str) else json_preview(data)
    print("ok", preview)

File: typescript/examples/smoke.ts
import { LkFloodApiDocsClient } from "../src/index.js";

/**
 * Live smoke — polite delay on. Run: npm run smoke
 * May fail if upstream is down / blocks this environment.
 */
async function main() {
  const client = new LkFloodApiDocsClient({ defaultDelayMs: 1000 });
  console.log("smoke", LkFloodApiDocsClient.slug, "->", "floodLatest");
  const data = await client.floodLatest();
  const preview = typeof data === "string" ? data.slice(0, 200) : JSON.stringify(data)?.slice(0, 200);
  console.log("ok", preview);
}

main().catch((err) => {
  console.error(err);
  process.exit(1);
});
